pkg/model/project.go

```diff
@@ -413,7 +413,7 @@ func (p *Project) AddUserGroup(sso, role string) error {
 	if p.HasUserGroup(sso) {
 		return fmt.Errorf("%s is already being used. The SSO group must be unique", sso)
 	}
-	if !p.HasRBACRole(role) {
+	if !p.HasRBACRole(role) && !isBuiltinRBACRole(role) {
```

Comment:
this looks like a leak to me, does it mean that there are some cases that `role` is not yet added into project RBAC roles but is being used as a role to add usergroup?

Reply:
The build-in role is not added to a project hence we have to check whether the role which is used by the user group is a built-in role or not when adding a user group.

Reply:
Ah get your point 👍 Then it's lgtm, thanks for the explanation.